contracts/hardhat.config.ts

```diff
@@ -1,6 +1,5 @@
+//import '@oasisprotocol/sapphire-hardhat';
```

Comment:
Leave it out then?

Reply:
Ah, my bad, I missed this while I was testing and merged hastily.

We should add an e2e test which verifies signed queries are working correctly.